pkg/cache/cacheconstants.go

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Since we don't currently have a `cache.go` file, I think this makes sense to go in a file with that name. Typically for a package, if you have some "common shared things" that apply to the whole package, this goes in a `*.go` file with that name.